emulsify-ds/compound
# feat: installed and configured webpack to do linting during dev

**Summary**

This PR fixes/implements the following **bugs/features**

- Installs and configures webpack to link during dev
- Fixes existing linting issues

## Steps to review

- Pull this branch
- `rm -rf node_modules`
- `nvm use`
- `npm install`
- `npm run develop`
	- Verify there are no errors in the console
- Edit the `01-atoms/tables/_tables.scss` file and delete the stylelint comments on lines 68 and 90
	- Verify you get stylelint errors in the console
- Edit the `02-molecules/menus/main-menu/main-menu.js` file and add `context = [];` somewhere in the file
	- Verify you get an eslint error in the console
